traffic_ops/app/bin/checks/DnssecRefresh/ToDnssecRefresh.go

```diff
@@ -0,0 +1,71 @@
+package main
+
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"github.com/apache/trafficcontrol/lib/go-log"
+	"github.com/apache/trafficcontrol/traffic_ops/app/bin/checks/DnssecRefresh/config"
+	"io/ioutil"
+	"net/http"
+	"net/http/cookiejar"
+	"os"
+)
+
+func main() {
+	cfg, err := config.GetCfg()
+	config.ErrCheck(err)
+	log.Debugln("Including DEBUG messages in output. Config is:")
+	config.PrintConfig(cfg) // only if DEBUG logging is set.
+	body := &config.Creds{
+		User:     cfg.TOUser,
+		Password: cfg.TOPass,
+	}
+	loginUrl := fmt.Sprintf("%s%s", cfg.TOUrl, "/api/2.0/user/login")
```

Comment:
Nitpick: might be easier to read as string concatenation, 
```
loginUrl := cfg.TOUrl + `/api/2.0/user/login`
```

Reply:
changed to string concatenation.